golem_messages/message/tasks.py

```diff
@@ -107,6 +108,9 @@ def __init__(self, header: datastructures.MessageHeader = None,
         if self.concent_enabled is None:
             self.concent_enabled = True
 
+        if isinstance(self.subtask_price, int):
+            assert self.subtask_price > 0
```

Comment:
1. Probably `deserialize_slot()` is a better place for verification.
1. Please use `FieldError` ( see: https://github.com/golemfactory/golem-messages/blob/9edd8179f2c9a9aedc00879f505e056dddd9631a/golem_messages/validators.py#L6-L12 )
1. Assertions are disabled in optimized mode and shouldn't be an integral part of production logic: https://docs.python.org/3/library/constants.html#__debug__

Reply:
Done, @jiivan please look at it one more time :)